novawallet/Modules/Staking/Validation/StakingDataValidatorFactory.swift

```diff
@@ -219,26 +219,55 @@ extension StakingDataValidatingFactory: StakingDataValidatingFactoryProtocol {
         })
     }
 
-    func stashIsNotKilledAfterUnbonding(
-        amount: Decimal?,
-        bonded: Decimal?,
-        minimumAmount: Decimal?,
+    func minStakeNotCrossed(
```

Comment:
Is there any reason we can't have StakingBaseValidatorFactory and have this method implementation once there? Currently it is duplicated in pools and direct staking validations as I understand.